int8 robotID
---
int8 pos_x
int8 pos_y
